### PR TITLE
kde-l10n fixes: New packages kdepim-l10n, ktp-l10n

### PR DESCRIPTION
**Pull request by a17r**
Introducing separate kde-apps/kdepim-l10n package similar to kde-base/kdepim-l10n. This solves missing KDE PIM 4 translations when using plasma-5/kde-apps-15.08 (see https://bugs.gentoo.org/show_bug.cgi?id=558944) while also solving ~~possible~~ file collisions between kde-apps/kde-l10n and kde-base/kdepim-l10n. Requires moving of kdepimlibs translations (4 and 5) into the respective kdepim-l10n packages.

### CODE REVIEW

File: kde-apps/kde-l10n/kde-l10n-15.08.0.ebuild [hunk 1]
```diff
@@ -8,7 +8,7 @@ KDE_HANDBOOK="true"
 inherit kde5
 
 DESCRIPTION="KDE internationalization package"
-HOMEPAGE="http://l10n.kde.org"
+HOMEPAGE="https://l10n.kde.org"
```
Comment (kensington) [inline kde-apps/kde-l10n/kde-l10n-15.08.0.ebuild]: Connection fails with https.

**Comment by a17r** [comment]
fwiw, it works fine here with kde4-l10n and kdepim-l10n (from this PR). I wouldn't expect otherwise using kde-l10n.

**Comment by a17r** [comment]
Split out another package from kde-l10n which shall henceforth be known as ktp-l10n. This is necessary as long as there are net-im/ktp-*:4 packages around which carry their own (colliding) translations. Those can block ktp-l10n now, while becoming compatible with kde-l10n.

**Comment by akhuettel** [comment]
reallyoldpim should not be a problem, just have a few untranslated strings

**Comment by a17r** [comment]
Latest commit also takes care of reallyoldpim's kdepim-l10n :) through a version bump, a test emerge didn't collide with installed kde-l10n, kde4-l10n.

**Comment by a17r** [comment]
Since I upgraded my main box to plasma-5.4, I also tested the German translations in KDE PIM 4 and it works for me.

**Pull request merged**